prov/psm2/Makefile.include

```diff
@@ -132,6 +135,7 @@ src_libfabric_la_LIBADD += libpsmx2.la
 src_libfabric_la_DEPENDENCIES += libpsmx2.la
 endif !HAVE_PSM2_DL
 
+rdmainclude_HEADERS += prov/psm2/include/fi_ext_psm2.h
```

Comment:
This causes make distcheck to fail.  See CI.

Reply:
The file was missing from the commit.  Fixed now.